### PR TITLE
feat(MNYe): Add MNYe basis trading system deploy scripts

### PR DESCRIPTION
**Pull request by 0xSachinK**
### [MNYe Prod Deployment](https://docs.google.com/document/d/1qgRAWxGgt3fCYBnhjVgOQaWurNM-lLE0Xb3kLqflbV0/edit#)

### CODE REVIEW

File: optimism/deployments/utils/dependencies.ts [hunk 1]
```diff
@@ -34,6 +34,15 @@ export default {
     69: "0x93910D1D112AEA86E20c998948897b0cd37B3Ed9"
   },
 
+  // Prod SetTokens
+  MNY_ETH_TOKEN: {
+    10: {
+      staging: "0x93910D1D112AEA86E20c998948897b0cd37B3Ed9",    // dummy address
```
Comment (0xSachinK) [inline optimism/deployments/utils/dependencies.ts:40]: @cgewecke I have to add a dummy address here for the tests to pass. Are there other ways?

File: optimism/deployments/constants/005_mny_eth_system.ts [hunk 2]
```diff
@@ -12,6 +12,8 @@ export const CONTRACT_NAMES = {
 
 export const IC_OPERATOR_MULTISIG = "0xA047F6900FeE0b6195088a49009eA65365f7DD68";
 
+export const ALLOWED_CALLER = "0x17d3FdB23128F8e376ffA444B8A84A30573864a6";
```
[the remark this answers is not in the document]
Comment (0xSachinK) [reply optimism/deployments/constants/005_mny_eth_system.ts:15]: @richardliang is the ALLOWED CALLER specific to this Set?

[the remark this answers is not in the document]
Comment (richardliang) [reply optimism/deployments/constants/005_mny_eth_system.ts:15]: yeah double check with Will, but that's our keeper address

**Pull request merged**